### DISCUSSION TITLE
Terms of Use

### DISCUSSION THREAD

**Discussion by moltra** [weather-gov/api#405] [General]
Is there a terms of use for the API?  Is there any restrictions on a commercial entity using this just to get data for an internal report / dashboard?

**Comment by StephenClouse** [comment]
All of the information presented via the API is intended to be open data, free to use for any purpose.

On a legal level, all of the data from API is either:

- Work prepared by an officer or employee of the United States federal government as part of that person's official duties, and is thus in the public domain under US copyright law;
- Data that NOAA has explicit permission to redistribute publicly and without restriction; or
- Bare facts that are not copyrightable

We probably should have an explicit ToU somewhere, though.

**Reply by fullychargedmarketing** [reply]
What about API usage limits?

**Reply by StephenClouse** [reply]
@fullychargedmarketing Unfortunately _that_ portion of the policy is above my pay grade.

**Reply by jah1007** [reply]
API shares rate limiting configuration with roughly 30 other NWS websites. Your limit will vary based on how many and how often you use them. While specific details are not shared, my experience has shown if you are using API and adhering to the most basic development principles you will not hit them. The rate limiting is done via IP so the only thing I would consider is if traffic coming out of your network has all the same IP (that is, it is being “natted” to the public).
